File: tuts/009-vpc-ipam-gs/php/Ec2Wrapper.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

namespace Example\Ec2;

use Aws\Ec2\Ec2Client;
use Aws\Exception\AwsException;

class Ec2Wrapper
{
    private Ec2Client $client;

    public function __construct(Ec2Client $client)
    {
        $this->client = $client;
    }

    // TODO: Add wrapper methods matching CLI tutorial actions
}

File: tuts/008-vpc-private-servers-gs/java/src/main/java/com/example/ec2/Ec2Wrapper.java
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package com.example.ec2;

import software.amazon.awssdk.services.ec2.Ec2Client;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class Ec2Wrapper {
    private static final Logger logger = LoggerFactory.getLogger(Ec2Wrapper.class);
    private final Ec2Client client;

    public Ec2Wrapper(Ec2Client client) {
        this.client = client;
    }

    // TODO: Add wrapper methods matching CLI tutorial actions
}
